src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -1025,7 +1033,7 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
       case ModuleName.percentageTransferManager:
         iface = new ethers.utils.Interface(PercentageTransferManager.abi);
         data = iface.functions.configure.encode([
-          (params.data as PercentageTransferManagerData).maxHolderPercentage.toNumber(),
+          valueToWei((params.data as PercentageTransferManagerData).maxHolderPercentage, new BigNumber(16)).toString(),
```

Comment:
You could create a constant for `new BigNumber(16)` as it will be always the number for percentages